workflow/sample_files/base-hvac-air-to-air-heat-pump-2-speed-research-features.xml

```diff
@@ -1,561 +0,0 @@
-<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
I think it's better to keep one test file for two speed systems, because the added staging logic will be in the place. If that kills the ci too much, we can consider reverting the backup increment input in the var speed test file (since it'll be tested in the 2-speed-research-features.xml), which I added [here](workflow/sample_files/base-hvac-air-to-air-heat-pump-var-speed-research-features.xml), also reverted the simulation timestep to 60mins for that var speed test file.

Reply:
@jmaguire1 Just to make you aware that we're reducing test files to speed up ci

Reply:
> I think it's better to keep one test file for two speed systems, because the added staging logic will be in the place. If that kills the ci too much, we can consider reverting the backup increment input in the var speed test file (since it'll be tested in the 2-speed-research-features.xml), which I added [here](workflow/sample_files/base-hvac-air-to-air-heat-pump-var-speed-research-features.xml), also reverted the simulation timestep to 60mins for that var speed test file.

Good point, I forgot that you have special controls for 2-speed systems relative to variable-speed systems. Please feel free to make any changes in this PR that you think are needed. There were originally [7 research feature HPXMLs](https://github.com/NREL/OpenStudio-HPXML/issues/1818), which I cut to 3 here. Ideally 3 would be enough, but if you really need 1 more for thorough testing, that's not the end of the world.